gradle.properties

```diff
@@ -0,0 +1,219 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name=apache-nutch
+version=1.19-SNAPSHOT
+final.name=apache-nutch-1.19-SNAPSHOT
+year=2021
+
+basedir = ./
+src.dir = ./src/java
+lib.dir = ./lib
+conf.dir = ./conf
+plugins.dir = ./src/plugin
+
+build.dir = ./build
+build.classes = ./build/classes
+build.plugins = ./build/plugins
+build.javadoc = ./build/docs/api
+build.encoding = UTF-8
+build.ivy.dir=./build/ivy
+build.lib.dir=./build/lib
+
+test.src.dir = ./src/test
+test.build.dir = ./build/test
+test.build.lib.dir = ./build/test/lib
+test.build.data =  ./build/test/data
+test.build.classes = ./build/test/classes
+test.build.javadoc = ./build/test/docs/api
+test.junit.output.format = plain
+
+# Proxy Host and Port to use for building JavaDoc
+javadoc.proxy.host=-J-DproxyHost=
+javadoc.proxy.port=-J-DproxyPort=
+javadoc.link.java=https://docs.oracle.com/en/java/javase/11/docs/api/
+javadoc.link.hadoop=https://hadoop.apache.org/docs/r3.1.3/api/
+#javadoc.link.lucene.core=https://lucene.apache.org/core/8_4_1/core/
+#javadoc.link.lucene.analyzers-common=https://lucene.apache.org/core/8_4_1/analyzers-common/
+#javadoc.link.solr-solrj=https://lucene.apache.org/solr/8_4_1/solr-solrj/
+javadoc.packages=org.apache.nutch.*
+
+dist.dir=./dist
+src.dist.version.dir=./dist/apache-nutch-1.19-SNAPSHOT-src
+bin.dist.version.dir=./dist/apache-nutch-1.19-SNAPSHOT-bin
+
+javac.debug=on
+javac.optimize=on
+javac.deprecation=on
+javac.version=11
+
+runtime.dir=./runtime
+runtime.deploy=./runtime/deploy
+runtime.local=./runtime/local
+
+ivy.version=2.5.0
```

Comment:
All of the ivy stuff can be removed. We don't need it anymore we are using Gradle :)